components/app.js

```diff
@@ -21,9 +24,11 @@ class App {
         grades.forEach(data => {
             gradeSum += data.grade;
         });
-        var gradeAverage = gradeSum / grades.length;
-
-        this.pageHeader.updateAverage(gradeAverage.toFixed(2));
+        if (gradeSum > 0) {
+            var gradeAverage = gradeSum / grades.length;
+            gradeAverage = Math.round((gradeAverage + Number.EPSILON) * 100) / 100

```

Comment:
What is `Number.EPSILON` doing for you here? This seems a bit over complicated for what the end goal is. Have you looked into [toFixed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed). Just curious what the rational is here?

Reply:
I used toFixed before this, but it added two decimal points even though it didn't need it. I  looked online for a way to do it without having the trailing decimal points and that's what I got. It did seem overkill. I looked a what it does but I'm not smart enough to understand. LOL.